### ISSUE TITLE
MinimumSecondsBetweenFailureNotifications does not work

### ISSUE THREAD

**Issue by wgv-rulak** [Xabaril/AspNetCore.Diagnostics.HealthChecks#321]
I was debbuging why this does not work . lastNotification 
 here always comes back as null , where conition has a bug. https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/d0b3804f7593b7fdcaf46e9acd5fb528a95e0f52/src/HealthChecks.UI/Core/Notifications/WebHookFailureNotifier.cs#L71-L75

if you remove the where and just do this it works, there may be some special characters in the healthchekname or livenessName
```
    var lastNotification = _db.Failures.ToList()
                .OrderByDescending(lf => lf.LastNotified)
                .FirstOrDefault();
```

**Comment by unaizorrilla** [comment]
Hi @CarlosLanderas 

Can you checkout this asap please?

**Comment by unaizorrilla** [comment]
Fixed on #322 ? A new release build is on appveyor, when finished a new package 3.0.1 will be on nuget. Please, close this if the issue is resolved.
